webcomponents-loader.js

```diff
@@ -154,8 +154,8 @@
 
     var newScript = document.createElement('script');
     newScript.src = url;
-    // if readyState is 'loading', this script is synchronous
-    if (document.readyState === 'loading') {
+    // if readyState is 'loading' and this script is synchronous
+    if (document.readyState === 'loading' && (!document.currentScript.async)) {
```

Comment:
Iirc, document.currentscript does not exist in type="module". Could you verify? (In that case, it would need to be `document.currentScript && ...`

Reply:
Fixed.